appengine-java11/cloudrun-pubsub/src/main/java/com/example/appengine/cloudrunpubsub/Body.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.appengine.cloudrunpubsub;
+
+// Body.Message is the payload of a Pub/Sub event. Please refer to the docs for
+// additional information regarding Pub/Sub events.
+public class Body {
```

Comment:
Usually a good idea to implement hash, compare, etc.  Not important for this, but in general it's a good practice.